client/src/components/frontend/ResourceCollection/Detail.js

```diff
@@ -13,6 +13,7 @@ export default class ResourceCollectionDetail extends PureComponent {
   static propTypes = {
     collection: PropTypes.object,
     project: PropTypes.object,
+    collectionUrl: PropTypes.string,
```

Comment:
Same comment as above—the collectionUrl should be required, or the share rendering should be conditional, I think.